package.json

```diff
@@ -42,5 +42,8 @@
     "jest-matcher-deep-close-to": "^3.0.2",
     "lerna": "^4.0.0",
     "prettier": "^2.6.1"
+  },
+  "dependencies": {
+    "ml-array-sum": "^1.1.6"
```

Comment:
Should not be at this level.

We are not using this dependency anymore. You should use https://github.com/mljs/spectra-processing/blob/master/src/x/xSum.ts